WcaOnRails/app/views/static_pages/about.html.erb

```diff
@@ -24,14 +24,13 @@
 
   <p><%= t('about.structure.committees') %>
   <ul>
-    <li><%= t('about.structure.wdc.name_html') %>: <%= format_team_members('wdc') %> <br />
-      <%= t('about.structure.wdc.description') %></li>
-    <li><%= t('about.structure.wrc.name_html') %>: <%= format_team_members('wrc') %> <br />
-      <%= t('about.structure.wrc.description') %></li>
-    <li><%= t('about.structure.results.name_html') %>: <%= format_team_members('results') %> <br />
-      <%= t('about.structure.results.description') %></li>
-    <li><%= t('about.structure.software.name_html') %>: <%= format_team_members('software') %> <br />
-      <%= t('about.structure.software.description') %></li>
+    <% [ "wct", "wdc", "wrc", "results", "software" ].each do |friendly_id| %>
+      <% team = Team.find_by_friendly_id!(friendly_id) %>
```

Comment:
Sorting teams becomes an issue if we want to do that. Sorting alphabetically by name could make sense, but not if we're translating the names. One solution would be to add a ranking field to the teams table to control the order the teams show up.